### PR TITLE
Bugfix/footer alignment

### PR DESCRIPTION
**Pull request by goyadev**
-Alignment woes fixed, open to feedback if you don't like it this way
- Some little easter eggs

Screenshot 2024-04-07 at 4 37 19 pm
Screenshot 2024-04-07 at 4 37 08 pm

### CODE REVIEW

File: src/main.jsx [hunk 1]
```diff
@@ -50,6 +52,9 @@ const router = createBrowserRouter([
       { path: "/aboutus", element: <AboutUs /> },
       { path: "/workshopideas", element: <AllWorkshopIdeasPage /> },
       { path: "/contactus", element: <ContactusPage /> },
+      // { path: "/privacy", element: <PrivacyPolicyPage /> },
```
Comment (goyadev) [inline src/main.jsx:55]: For the life of me I couldn't work out why I couldn't get this pathway to work. open to suggestions!

File: src/components/Footer.jsx [hunk 2]
```diff
@@ -4,40 +4,42 @@ import FacebookLogo from "../assets/icons/facebook-2.svg";
...
+            <a href="/privacy" className="hover:text-white">
               Privacy Policy
             </a>
-            <a href="/" className="hover:text-secondary">
+            <a href="/termsandconditions" className="hover:text-white">
               Terms & Conditions
             </a>
-            <a href="/" className="hover:text-secondary">
+            <a
+              href="https://youtu.be/Ye8mB6VsUHw?si=t8GPJxh3FpjzYYim&t=10"
```
Comment (suanTech) [inline src/components/Footer.jsx:38]: Got ya!!

**Review by suanTech — COMMENTED** [review]
I think the socials could be aligned on top of the links like this ??

Screenshot 2024-04-07 at 4 42 36 pm

Screenshot 2024-04-07 at 4 43 45 pm

File: src/main.jsx (hunk 1 above)
Comment (suanTech) [reply src/main.jsx:55]: what was the error?

Comment (goyadev) [reply src/main.jsx:55]: the whole app wouldn't render. white page of death

**Comment by goyadev** [comment]
Updated placement of footer elements

Screenshot 2024-04-08 at 3 51 14 pm
Screenshot 2024-04-08 at 3 51 03 pm
Screenshot 2024-04-08 at 3 50 55 pm

**Review by suanTech — APPROVED** [review]

**Pull request merged**